README.md

```diff
@@ -1 +1,37 @@
-# my-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## My Portfolio
+This is an HTML psge wich content Ahmad Zaki Yousufi's portfolio.
+
+## Built With
+HTML
+CSS
+Getting Started
+## To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+A browser
+
+Usage
+Clone the repositry and open the index.html which is in the folder.
+
+## Authors
+GitHub: @zakiyousufi
+Twitter: @ahmadzakiyousofi
+LinkedIn: ahmadzakiyousufi
```

Comment:
**[REQUIRED  CHANGES]**

Please ensure your social media account is a link that takes a user to the referred account.